src/Entity/CommitteeCandidacy.php

```diff
@@ -0,0 +1,107 @@
+<?php
+
+namespace AppBundle\Entity;
+
+use Doctrine\ORM\Mapping as ORM;
+use Gedmo\Timestampable\Traits\TimestampableEntity;
+
+/**
+ * @ORM\Entity(repositoryClass="AppBundle\Repository\CommitteeCandidacyRepository")
+ */
+class CommitteeCandidacy
+{
+    use TimestampableEntity;
+
+    /**
+     * @ORM\Id
+     * @ORM\Column(type="integer")
+     * @ORM\GeneratedValue(strategy="AUTO")
+     */
+    private $id;
+
+    /**
+     * @var \DateTimeImmutable|null
+     *
+     * @ORM\Column(type="datetime_immutable", nullable=true)
+     */
+    private $removedAt;
+
+    /**
+     * @var CommitteeMembership
+     *
+     * @ORM\OneToOne(targetEntity="CommitteeMembership")
+     * @ORM\JoinColumn(nullable=false)
+     */
+    private $committeeMembership;
+
+    /**
+     * @var CommitteeElection
+     *
+     * @ORM\ManyToOne(targetEntity="CommitteeElection")
+     * @ORM\JoinColumn(nullable=false)
```

Comment:
`@ORM\JoinColumn(nullable=false, onDelete="CASCADE")`
=> this will tell the database to automatically remove this `CommitteeCandidacy` when the related `CommitteeElection` is deleted